proto/storage/vulnerability.proto

```diff
@@ -40,14 +40,14 @@ message EmbeddedVulnerability {
     google.protobuf.Timestamp  last_modified           = 10;
     // For internal purposes only.
     VulnerabilityType          vulnerability_type      = 11;
-    repeated VulnerabilityType vulnerability_types     = 18;
+    repeated VulnerabilityType vulnerability_types     = 18 [(gogoproto.moretags) = 'hash:"ignore"'];
```

Comment:
Just want to hear your thoughts on the following:

Since this proto is embedded it is always (meant to) present in the context of workload or host. Thus, these fields can be removed. I am thinking put a deprecation notice for these fields in 4.3.

Reply:
Yeah I think we should deprecate this field